packaging/chocolatey/crc/tools/chocolateyuninstall.ps1

```diff
@@ -0,0 +1,11 @@
+﻿$ErrorActionPreference = 'SilentlyContinue';
+$toolsDir   = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$setupScript = Join-Path "$toolsDir" "crcprerequisitesetup.ps1"
+
+Import-Module $setupScript
+
+# remove crc-users group
+Remove-LocalGroup -Name 'crc-users' -Confirm:$false | Out-Null
+
+# remove vsock registry element
+Remove-VsockGuestCommunicationRegistry | Out-Null

```

Comment:
Isn't this similar to hyperv, we can't know if the vsock key was created by crc or not, so we should not remove it?

Reply:
yeah right, also the check for existence of this needs to be improved currently its checking if the `ElementName` property exists, but we should check also the value of that property is `gvisor-tap-vsock`

Reply:
not removing this registry key now